src/wallet/wallet.h

```diff
@@ -428,6 +428,11 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
     //! Cache of descriptor ScriptPubKeys used for IsMine. Maps ScriptPubKey to set of spkms
     std::unordered_map<CScript, std::vector<ScriptPubKeyMan*>, SaltedSipHasher> m_cached_spks;
 
+    /**
+     * Tracks txids of unrelated double-spending txs  to wallet transactions
```

Comment:
In 7eba56accbd1b4ce3c09656c8d69b5aa437a706f "Handle double-spending of unrelated parents to wallet txs"

nit: whitespace.

Also, I think it would be better to clarify that the key is the txid of the replacing transaction which may be unrelated to the wallet, and the value is the replaced transaction which is related to this wallet.

Reply:
Fixed in https://github.com/bitcoin/bitcoin/pull/29680/commits/3c893c51426b2bd9b9b504d00614f3f1742322d5